docs/engine/README.md

```diff
@@ -24,3 +24,14 @@ If the Engine finds resources that have the tag set to `True`, it then evaluates
     }
 }
 ```
+
+## Adjusting the Bellhop Engine Interval
+
+The interval at which the Bellhop engine runs is set by the Application Setting 'ENGINE_TIMER_EXPRESSION'
+
+![Bellhop Interval Setting](../images/timer_interval_setting.png)
+
+- The Application Settings for the Function can be found here
+    - &lt;deployment prefix&gt;-rg 🠆 &lt;deployment prefix&gt;-function-engine 🠆 Configuration 🠆 'ENGINE_TIMER_EXPRESSION'
```

Comment:
The "-->" don't show up for me here. Maybe there is another way to show this?

Maybe:
- &lt;deployment prefix&gt;-rg --> &lt;deployment prefix&gt;-function-engine --> Configuration --> 'ENGINE_TIMER_EXPRESSION'

Reply:
Looks good in the docs now, thanks @DCMattyG

Reply:
Updated the arrow character in the documentation, they should work more universally now.